compiler/resolution/functionResolution.cpp

```diff
@@ -2413,6 +2413,12 @@ printResolutionErrorUnresolved(
                      Vec<FnSymbol*>& visibleFns,
                      CallInfo* info) {
   CallExpr* call = userCall(info->call);
+
```

Comment:
I think this is fine for now.
If "developers" find it annoying, we can add the new flag as needed.